### PR TITLE
Remove standalone references from API integration tests files

### LINKED ISSUE #13764: Remove standalone option in API integration tests
## Description

The Wazuh cluster will always be enabled and running after the changes in https://github.com/wazuh/wazuh/issues/7108. As a consequence, the `standalone` option in the API integration tests should be removed. 

It is necessary to delete all the related code and check that the tests still work after that. For instance:
https://github.com/wazuh/wazuh/blob/4ad7e9cddfb31b5e0aea978332d2185f00d94f4c/api/test/integration/env/base/manager/entrypoint.sh#L11-L13

### wazuh/wazuh
- Unit tests without failures. Updated and/or expanded if there are new functions/methods/outputs:
  - [x] Cluster (`framework/wazuh/core/cluster/tests/` & `framework/wazuh/core/cluster/dapi/tests/`)
  - [x] Core (`framework/wazuh/core/tests/`)
  - [x] SDK (`framework/wazuh/tests/`)
  - [x] RBAC (`framework/wazuh/rbac/tests/`)
  - [x] API (`api/api/tests/`)
- API tavern integration tests without failures. Updated and/or expanded if needed (`api/test/integration/`):
  - [x] Affected tests 
  - [x] Affected RBAC (black and white) tests
- [x] Review integration test mapping using the script (`api/test/integration/mapping/integration_test_api_endpoints.json`)
- [x] Review of spec.yaml examples and schemas (`api/api/spec/spec.yaml`)
- [x] Review exceptions remediation when any endpoint path changes or is removed (`framework/wazuh/core/exception.py`)
- [x] Changelog (`CHANGELOG.md`)

### wazuh/wazuh-documentation
- [x] Migration from 3.X for changed endpoints (`source/user-manual/api/equivalence.rst`)
- [x] Update RBAC reference with new/modified actions/resources/relationships (`source/user-manual/api/rbac/reference.rst`)

### PR DESCRIPTION
**Pull request by vicferpoy**
|Related issue|
|---|
|closes #13764 |

## Description

This PR removes every reference to the standalone AIT from the code, tools and documentation.

## Tests performed

```
(ait-env) vicferpoy:~/Desktop/Git/wazuh/api/test/integration (feature/13764-remove-standalone-ait)$ python3 run_tests.py -r
test_cluster_endpoints
	 45 passed, 57 warnings

test_manager_endpoints
	 1 failed, 30 passed, 33 warnings

```
> The failing test is not related to this issue.

Regards,
Víctor

### CODE REVIEW

File: api/test/integration/run_tests.py [hunk 1]
```diff
@@ -139,9 +129,7 @@ def run_test(test: str, iteration: int, pytest_mark: str = None):
             Extra mark used to pass the API integration test in a cluster or standalone environment.
         """
         test_name = \
-            f'{test.rsplit(".")[0]}' \
-            f'{STANDALONE_SUFFIX if pytest_mark == PYTEST_MARK_STANDALONE else CLUSTER_SUFFIX if pytest_mark == PYTEST_MARK_CLUSTER else ""}' \
-            f'{iteration if iteration != 1 else ""}'
+            f'{test.rsplit(".")[0]}' f'{iteration if iteration != 1 else ""}'
```
Comment (mcarmona99) [inline api/test/integration/run_tests.py:132]: This can be written in one line.

File: api/test/integration/run_tests.py [hunk 2]
```diff
@@ -187,14 +158,8 @@ def get_results(filename: str = None):
...
-        for file in sorted(glob.glob('test_*_standalone*')):
-            print(f"{file} - Standalone environment")
-            calculate_result(file)
-        for file in sorted(glob.glob('test_*_cluster*')):
-            print(f"{file} - Cluster environment")
-            calculate_result(file)
-        for file in sorted(glob.glob('test_rbac*')):
-            print(file)
+        for file in sorted(glob.glob('test_*')):
+            print(f"{file}")
```
Comment (mcarmona99) [inline api/test/integration/run_tests.py:162]: No need to use f-string
```suggestion
            print(file)
```

File: api/test/integration/run_tests.py [hunk 3]
```diff
@@ -139,9 +129,7 @@ def run_test(test: str, iteration: int, pytest_mark: str = None):
             Extra mark used to pass the API integration test in a cluster or standalone environment.
         """
         test_name = \
-            f'{test.rsplit(".")[0]}' \
-            f'{STANDALONE_SUFFIX if pytest_mark == PYTEST_MARK_STANDALONE else CLUSTER_SUFFIX if pytest_mark == PYTEST_MARK_CLUSTER else ""}' \
-            f'{iteration if iteration != 1 else ""}'
+            f'{test.rsplit(".")[0]}' f'{iteration if iteration != 1 else ""}'
         html_params = [f"--html={RESULTS_FOLDER}/html_reports/{test_name}.html", '--self-contained-html']
         with open(os.path.join(RESULTS_FOLDER, test_name), 'w') as f:
```
Comment (mcarmona99) [inline api/test/integration/run_tests.py:134]: Review this block, `pytest_mark` is never used.

The `run_test` function was created to make the code clearer after the addition of the standalone mode. We could consider avoiding this function now. Have a look at the changes done on https://github.com/wazuh/wazuh/pull/12360.

**Review by mcarmona99 — CHANGES_REQUESTED** [review]
After the changes, `import subprocess` is not necessary in `api/test/integration/env/tools/healthcheck_utils.py` 

Also, in that same file, the `CHECK_CLUSTERD_DAEMON` variable is not needed.

Review `test_agent_GET_endpoints.tavern.yaml`

**Review by mcarmona99 — APPROVED** [review]
LGTM

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**